services/supervisor/tests/simple.rs

```diff
@@ -154,11 +169,15 @@ fn exclude_us_from_validators() {
 
     let old_validators = testkit.network().validators();
 
-    testkit.create_block_with_transaction(
-        ConfigPropose::actual_from(cfg_change_height)
-            .consensus_config(new_consensus_config.clone())
-            .sign_for_simple_supervisor(),
-    );
+    let initiator_id = testkit.network().us().validator_id().unwrap();
+    let config_propose = ConfigPropose::actual_from(cfg_change_height)
+        .consensus_config(new_consensus_config.clone());
+
+    testkit.create_block_with_transaction(sign_config_propose_transaction(
```

Comment:
Nit: Maybe, it could make sense to implement a variation of `sign_config_propose_transaction` in which `validator_id` is set as `testkit.network().us().validator_id().unwrap()`. This would reduce the amount of repeated code a little bit.